luaui/Widgets/Shaders/map_startcone_gl4.vert.glsl

```diff
@@ -0,0 +1,51 @@
+//map_startcone_gl4.vert.glsl
+
+#version 420
+#extension GL_ARB_uniform_buffer_object : require
+#extension GL_ARB_shader_storage_buffer_object : require
+#extension GL_ARB_shading_language_420pack: require
+// This shader is (c) Beherith (mysterme@gmail.com), licensed under the MIT license
+
+#line 5000
+layout (location = 0) in vec4 position; // xyz and etc garbage
+//layout locations 1 and 2 contain primitive specific garbage and should not be used
+layout (location = 3) in vec4 worldposrad; // l w rot and maxalpha
+layout (location = 4) in vec4 teamcolor;
+
+//__ENGINEUNIFORMBUFFERDEFS__
+//__DEFINES__
+
+#line 10000
+
+uniform float isMinimap = 0;
+uniform int flipMiniMap = 0;
+
+out DataVS {
+	vec4 v_worldposrad;
```

Comment:
spacing here all over the place... tabs, spaces, these ones look wrong too: `}else{`, `vec2    ndcxy`, `- ndcxy.y`